examples/react-native-todo-list/src/components/WhoIsHere.tsx

```diff
@@ -0,0 +1,80 @@
+import React from "react";
+import { StyleSheet, View, Text } from "react-native";
+
+import Avatar, { MoreAvatars } from "../../assets/svg/Avatars";
+
+const colorPairs = [
+  {
+    startColor: "#002A95",
+    endColor: "#00A0D2",
+    id: 1,
+  },
+  {
+    startColor: "#6116FF",
+    endColor: "#E32DD1",
+    id: 2,
+  },
+  {
+    startColor: "#39C7D1",
+    endColor: "#62CC52",
+    id: 3,
+  },
+];
+
+export type AvatarsTypeProps = {
+  count: number;
+};
+
```

Comment:
Feel redundant... any thoughts?

Reply:
I don't think it's redundant. It's two components with the same props, it's okay to be explicit.